src/compute/generic.ts

```diff
@@ -100,15 +103,18 @@ export async function computeTermAggregationByYear(
     db: Db,
     survey: SurveyConfig,
     key: string,
-    options: TermAggregationByYearOptions = {}
+    options: TermAggregationByYearOptions = {},
+    year?: number
```

Comment:
Yes, I did it this way because I didn't want the function to sometimes return an array, and sometimes return a single element. But we could have two functions.

Reply:
You're right, consistency is better, I guess we can just "wrap" this in a different function to return a single year then.